src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User userX = userService.findByEmail(email);
+        if (userX != null && userX.getPassword().equals(password)){
+            return true;
+        }
         return false;
```

Comment:
Also, you can simplify  it 
```suggestion
        return user != null && userX.getPassword().equals(password);
```